mobile-app/app/components/SellComponents/BottomSheetFiatAccountList.tsx

```diff
@@ -83,32 +83,34 @@ export const BottomSheetFiatAccountList = ({
           header: () => null
         }
       }])
-  }, [fiatAccounts])
+  }, [bankAccounts])
 
   return (
     <>
       <FlatList
-        data={accountList}
+        data={accountList as any}
         extraData={trigger}
-        renderItem={({ item }: { item: SellRoute }): JSX.Element => {
+        renderItem={({ item }: { item: BankAccount }): JSX.Element => {
           return (
             <ThemedTouchableOpacity
               onPress={() => {
-                if (onFiatAccountPress !== undefined) {
-                  onFiatAccountPress(item)
-                }
+                onFiatAccountPress?.(item)
               }}
               style={tailwind('px-4 py-3 flex flex-row items-center justify-between')}
               testID={`select_${item.iban}`}
             >
-              <View style={tailwind('flex flex-row items-center')}>
-                <View style={tailwind('ml-2')}>
-                  <ThemedText
-                    testID={`token_symbol_${item.iban}`}
-                  >
-                    {`${item.fiat.name} / ${item.iban}`}
-                  </ThemedText>
-                </View>
+              <View style={tailwind('ml-2')}>
+                {item.sepaInstant && (
+                  <View style={tailwind('flex-shrink flex-row')}>
+                    <SepaInstantComponent red invertedColor />
+                  </View>
+                )}
+                <ThemedText
+                  testID={`token_symbol_${item.iban}`}
+                >
+                  {/* {`${item?.label ?? (item?.fiat?.name ? item.fiat.name + ' / ' : ' ')}${item.iban}`} */}
```

Comment:
comment can be removed, or is it still up for discussion if another text formatting should be used